scripts/ci-e2e.sh

```diff
@@ -70,6 +70,43 @@ source "${M3_DEV_ENV_PATH}/lib/ironic_tls_setup.sh"
 # image for live iso testing
 export LIVE_ISO_IMAGE="https://artifactory.nordix.org/artifactory/metal3/images/iso/minimal_linux_live-v2.iso"
 
+# Generate credentials
+BMO_OVERLAYS=("${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-0.3" "${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-0.4" "${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-0.5" "${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-0.6")
+IRONIC_OVERLAYS=("${REPO_ROOT}/test/e2e/data/ironic-deployment/overlays/release-23.1" "${REPO_ROOT}/test/e2e/data/ironic-deployment/overlays/release-24.0" "${REPO_ROOT}/test/e2e/data/ironic-deployment/overlays/release-24.1")
+
+IRONIC_USERNAME="test-user"
```

Comment:
```suggestion
IRONIC_USERNAME="$(uuidgen)"
```
maybe do the same as BMO e2e, use uuidgen?

Reply:
fixed